### PR TITLE
[TASK] Update the development tools

### PR DESCRIPTION
**Pull request by oliverklee**
Also update the Psalm baseline. (We can fix the new issues in a separate
change.)

### CODE REVIEW

**Comment by oliverklee** [comment]
I have entered #1107 for the formatting issue.

File: tests/Unit/Support/Constraint/IsEquivalentCssTest.php [hunk 1]
```diff
@@ -16,6 +16,7 @@
 final class IsEquivalentCssTest extends TestCase
 {
     use CssDataProviders;
+
```
Comment (JakeQZ) [inline tests/Unit/Support/Constraint/IsEquivalentCssTest.php:19]: I saw that the blank line issue with PHP-CS-Fixer is being dealt with separately.

File: tests/Unit/Support/Constraint/StringContainsCssCountTest.php [hunk 2]
```diff
@@ -16,6 +16,7 @@
 final class StringContainsCssCountTest extends TestCase
 {
     use CssDataProviders;
+
```
Comment (JakeQZ) [inline tests/Unit/Support/Constraint/StringContainsCssCountTest.php:19]: Ditto.

File: tests/Unit/Support/Constraint/StringContainsCssTest.php [hunk 3]
```diff
@@ -16,6 +16,7 @@
 final class StringContainsCssTest extends TestCase
 {
     use CssDataProviders;
+
```
Comment (JakeQZ) [inline tests/Unit/Support/Constraint/StringContainsCssTest.php:19]: Ditto.

**Review by JakeQZ — APPROVED** [review]
Approved on the basis that the issue with PHP-CS-Fixer requiring blank lines between `use` statements is being covered separately.

**Pull request merged**